Libraries/oneMKL/fourier_correlation/sample.json

```diff
@@ -0,0 +1,32 @@
+{
+  "guid": "69F6B0A1-3046-467F-B4A7-E61F636BC5BF",
+  "name": "Fourier Correlation",
```

Comment:
I updated the readme and sample.json file, please inspect and verify I didn't change your message too much

Reply:
These changes and fixes are fine.